.gitignore

```diff
@@ -126,3 +126,4 @@ celerybeat-schedule
 docker-compose.*.yml
 
 .python-version
+/locustfile.py
```

Comment:
This is just so that when we DO add locustfiles, they don't accidentally go to prod - locust won't run without one, so we won't accidentally... ddos production somehow?